### ISSUE TITLE
SSM Provider Example Config File

### ISSUE THREAD

**Issue by kevinswarner** [CoorpAcademy/serverless-plugins#105]
It is not clear what format the file/contents of the file for the SSM values needs to be. Can you provide an example?

**Comment by kevinswarner** [comment]
I figured it out be looking through source. It might be helpful to have an example on the docs page for this plugin though. Thanks! For anyone else looking, you simply include the keys and values as you would with a .env file. For example, if you have the following reference in the serverless.yml file...

${ssm:/foo/bar/baz}

You would put a line in the .ssm file like this...

/foo/bar/baz=myvaluehere

So, it is...

{SSM_PATH}={VALUE}

**Comment by AdrieanKhisbe** [comment]
Hello @kevinswarner, sorry for this far too late answer. :/

it's indeed a `$path=$value` pattern [cf integration tests](https://github.com/CoorpAcademy/serverless-plugins/blob/master/tests/serverless-plugins-integration/.ssm#L1-L4)

```
/sqs/port=9324
/s3/port=9000
/dynamodb/port=8000
/kinesis/port=4567
```

**Comment by silouone** [comment]
Good call-out @kevinswarner — the expected file is a flat YAML/JSON map of SSM parameter paths to values (e.g. `/my-service/dev/DB_URL: postgres://...`) that `serverless-offline-ssm-provider` resolves in place of real SSM. We'll make this clearer in the docs; leaving this open as a documentation request. 🙏
